lib/govuk_test.rb

```diff
@@ -11,10 +11,16 @@ def self.configure(options = {})
     chrome_options << "--window-size=#{options[:window_size]}" if options[:window_size]
 
     if ENV['GOVUK_TEST_USE_SYSTEM_CHROMEDRIVER']
-      # Use the installed chromedriver, rather than chromedriver-helper
+      # Use the installed chromedriver, rather than webdrivers
       Selenium::WebDriver::Chrome.driver_path = File.which("chromedriver")
     else
-      require 'chromedriver-helper'
+      require 'webdrivers'
+      if Gem.loaded_specs.has_key?('webmock')
+        WebMock.disable_net_connect!({
```

Comment:
I think you don't need the curly brackets here as Ruby will consider it as hash already.

Reply:
+1+